packages/mcdoc/src/binder/index.ts

```diff
@@ -526,16 +539,22 @@ async function bindPath(
 				'mcdoc',
 				pathArrayToString(identifiers),
 			)
-			.ifDeclared((_, query) =>
-				query.enter({
-					usage: {
-						type: 'reference',
-						node: identNode,
-						fullRange: node,
-						skipRenaming: LiteralNode.is(identNode),
-					},
-				})
-			)
+			.ifDeclared((_, query) => {
+				try {
+					query.resolveAlias().enter({
+						usage: {
+							type: 'reference',
+							node: identNode,
+							fullRange: node,
+							skipRenaming: LiteralNode.is(identNode),
+						},
+					})
+				} catch (_ignored) {
```

Comment:
(nitpick)

```suggestion
				} catch {
```

if we're already explaining why we do nothing with the error i don't see the need to grab the parameter

Reply:
didn't know you could do that